<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Play</class>
 <widget class="QMainWindow" name="Play">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1601</width>
    <height>801</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonSensitivity">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>350</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Sensitivity</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonIdentifiability">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>390</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Identifiability</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>291</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Core Settings</string>
    </property>
    <widget class="QComboBox" name="comboBoxExperiment">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>80</y>
       <width>161</width>
       <height>61</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxDataSet">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>200</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>53</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Model</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxModele">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Data Set</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ExperimentSet</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Experiment</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxSubExperiment">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>150</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>661</width>
      <height>371</height>
     </rect>
    </property>
    <property name="title">
     <string>Parameter Work ...</string>
    </property>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>631</width>
       <height>291</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonLoadOpt">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>330</y>
       <width>131</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Optimizer File</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSaveSet">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>330</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Save Set</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonLoadSet">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>330</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Set</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>390</y>
      <width>661</width>
      <height>341</height>
     </rect>
    </property>
    <property name="title">
     <string>Visualisation</string>
    </property>
    <widget class="QComboBox" name="comboBoxVariable">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>53</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Variable</string>
     </property>
    </widget>
    <widget class="QWidget" name="widgetGraphe" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>631</width>
       <height>281</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>10</y>
      <width>581</width>
      <height>581</height>
     </rect>
    </property>
    <property name="title">
     <string>Working Files</string>
    </property>
    <widget class="QTextEdit" name="textEditOptimizerFile">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>541</width>
       <height>191</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>311</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Optimizer file header</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>311</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Current parameter set</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEditParamSet">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>270</y>
       <width>541</width>
       <height>61</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEditOptimizerFileFull">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>360</y>
       <width>541</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>340</y>
       <width>311</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Current optimizer file</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButtonGo">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>270</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Simulate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOptimize">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>310</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Optimize</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1601</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
